android-ktx/build.gradle

```diff
@@ -35,7 +35,7 @@ configurations.all {
 
 dependencies {
     api "com.mirego.trikot:streams:$trikot_streams_version"
-    api 'com.mirego.trikot:http:0.22.1'
+    api 'com.mirego.trikot:http:0.24.1'
```

Comment:
Looks like this is preventing the build to pass

```
Could not find com.mirego.trikot:http:0.24.1.
```

The latest version is `0.23.1`.

Reply:
This is "normal" I did the changes in android-ktx that are dependent on the release of 0.24.1. It's the chicken or the egg problem here. If we don't wan't to do this, I would have to do 2 PR one for http and the other for ktx but the files are in the same repo... 🤔

Reply:
Yes, but usually feature pull request do not increase the version.

Is this because @mirego-builds automatically increase the version when a commit is merged in `master`? If this is the case, it should also update the `android-ktx/build.gradle` file, right?

Reply:
you could want to have a different version of http in ktx son that's why it's like this. Next time i'll do the work in 2 separate PR to avoid confusion.